server/package.json

```diff
@@ -74,7 +74,8 @@
     "reflect-metadata": "^0.1.13",
     "rxjs": "^7.2.0",
     "sanitize-filename": "^1.6.3",
-    "sharp": "^0.31.0",
+    "sharp": "^0.31.3",
+    "thumbhash-node": "^0.1.1",
```

Comment:
Also, shouldn't we use the official thumbhash package rather than an unmaintained third party package?

https://www.npmjs.com/package/thumbhash

Reply:
I chose this lib because immich uses CommonJS while the official `thumbhash` package is an ESM module. I was having difficulty getting it to import properly so I gave up and used a 3rd party package.

There likely is a way to make it work properly, but I didn't see it as a big issue as this package is kinda a "if it works it works" package and we don't really have to worry about it being maintained. 

Also, in my view, it being a rust binding is good. Generally rust library's are far less computationally heavy.

Reply:
I'm concerned that using a third party package will not receive upstream updates if required and don't think we should add unnecessary complexity to Immich. Using a rust binding has all sorts of strangeness like [pre-compiled platform binaries](https://github.com/amehashi/thumbhash-node/tree/2c2aff8bc30d9bd96a7cd100f0600379690591bc/npm). It would be much better if we could someone use the official package as it's much simpler.

Reply:
Fair enough. Alex suggested another way I could try to get the official package to work so I'll give that a shot.